management_api_app/api/routes/workspaces.py

```diff
@@ -93,20 +101,61 @@ async def create_workspace_service(workspace_create: WorkspaceServiceInCreate,
     return WorkspaceServiceIdInResponse(workspaceServiceId=workspace_service.id)
 
 
+@router.post("/workspaces/{workspace_id}/workspace-services/{service_id}/user-resources",
+             status_code=status.HTTP_202_ACCEPTED,
+             response_model=UserResourceIdInResponse, name=strings.API_CREATE_USER_RESOURCE)
+async def create_user_resource(user_resource_create: UserResourceInCreate,
+                               user_resource_repo: UserResourceRepository = Depends(
+                                   get_repository(UserResourceRepository)),
+                               user: User = Depends(get_current_user),
+                               workspace: Workspace = Depends(get_deployed_workspace_by_workspace_id_from_path),
+                               workspace_service: WorkspaceService = Depends(get_deployed_workspace_service_by_id_from_path)
+                               ) -> UserResourceIdInResponse:
+    access_service = get_access_service()
+    role = access_service.get_workspace_role(user, workspace)
+    if (role != WorkspaceRole.Researcher) and (role != WorkspaceRole.Owner):
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail=strings.ACCESS_USER_IS_NOT_OWNER)
+
+    try:
+        user_resource = user_resource_repo.create_user_resource_item(user_resource_create, workspace.id,
+                                                                     workspace_service.id)
+    except (ValidationError, ValueError) as e:
+        logging.error(f"Failed create workspace service model instance: {e}")
+        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
+
+    try:
+        user_resource_repo.save_resource(user_resource)
+    except Exception as e:
+        logging.error(f"Failed save user resource instance in DB: {e}")
+        raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+                            detail=strings.STATE_STORE_ENDPOINT_NOT_RESPONDING)
+
+    try:
+        await send_resource_request_message(user_resource, RequestAction.Install)
+    except Exception as e:
+        # TODO: Rollback DB change, issue #154
```

Comment:
#154 is now fixed in a pending PR. Please change here accordingly when that PR is in